pkgs/tools/networking/passh/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "passh";
+  version = "2019-07-04";
+
+  src = fetchFromGitHub {
+    owner = "clarkwang";
+    repo = pname;
+    rev = "f1cc1403d29aa90128743cfac84ba61820a256e8";
+    sha256 = "147cj4045yrwxrb0kz4pll2y40q9j82g5ibcibvx7q5833jmfydh";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp passh $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/clarkwang/passh";
+    description = "sshpass alternative for non-interactive ssh auth";
+    license = licenses.unfree; # Author does not set a license
```

Comment:
Please file an issue with them asking them to issue a license? I don't know if we can package it if there is literally no license. Does GitHub's base license grant enough permission to run the code?

Reply:
https://github.com/clarkwang/passh/issues/5

Reply:
@grahamc the author kindly licensed it as GPLv3, we're good to go!